### PR TITLE
@mr0sari0 form2346 schema setup chapter 1

### PR DESCRIPTION
**Pull request by maharielrosario**
## Description
JSON/ UI schema setup for chapter 1 of Form-2346

## Screenshots
screenshot-localhost_3001-2020 02 (5)

screenshot-localhost_3001-2020 02 (2)

screenshot-localhost_3001-2020 02 (3)

screenshot-localhost_3001-2020 02 (4)

## Acceptance criteria
- [ ] Data is able to be shown on the review page
- [ ] UI schema is displaying elements correctly
- [ ] JSON schema and UI schemas are in sync

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by LinnJS** [comment]
This is what I am talking about, very clean. very easy to follow, and best part of all not that much code which make it very maintainable. Love how you put all your content in one folder to make content edits easier, also makes integrating with a data provider that much easier cause all the variables will be in one organized file.

**Review by mojaray2k — APPROVED** [review]

**Comment by mojaray2k** [comment]
> This is what I am talking about, very clean. very easy to follow, and best part of all not that much code which makes it very maintainable. Love how you put all your content in one folder to make content edits easier, also makes integrating with a data provider that much easier cause all the variables will be in one organized file.

Hey @LinnJS can you approve this please. Another ticket is dependent on this.

**Review by LinnJS — APPROVED** [review]

File: src/applications/disability-benefits/2346/config/form.js [hunk 1]
```diff
@@ -1,22 +1,38 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-sequences */
```
Comment (bkjohnson) [inline src/applications/disability-benefits/2346/config/form.js]: Why are you disabling these ESLint rules?

Comment (mr0sari0) [reply src/applications/disability-benefits/2346/config/form.js]: Just removed these comments.

**Comment by mr0sari0** [comment]
@bkjohnson Hi Brooks, I just applied your review. Please approve if you don't see anything else.

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**